opentracing/tracer.go

```diff
@@ -0,0 +1,112 @@
+package opentracing
+
+import (
+	"time"
+
+	opentracing "github.com/opentracing/opentracing-go"
+	"sourcegraph.com/sourcegraph/appdash"
+)
+
+var (
+	// The zero value of an unitialized time.Time
+	timeZeroValue = time.Time{}
+)
+
+// Tracer is the appdash implementation of the opentracing-go API.
+type Tracer struct {
+	recorder         *appdash.Recorder
+	options          Options
+	textPropagator   *splitTextPropagator
+	binaryPropagator *splitBinaryPropagator
+	goHTTPPropagator *goHTTPPropagator
+}
+
```

Comment:
Should add some comments. The idea was taken from standard tracer. It should be used to determine the sampling status of a function, ie `func(traceID int64) { return traceID % 1024 }`